client/scss/overrides/_vendor.tippy.scss

```
@@ -29,3 +29,17 @@
     @apply w-p-0;
   }
 }
+
+// Media for Windows High Contrast mode
+@media (forced-colors: $media-forced-colours) {
+  .tippy-box[data-theme='dropdown'] {
+    .tippy-content {
+      border: 2px solid;
```

```suggestion
      border: 2px solid transparent;
```

Only minor tweak is to use transparent here, this way it gets a border colour, but the WHCM will convert this to a suitable colour.

---

Thanks LB!